modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -307,10 +307,26 @@ public void processOpts() {
                 }
             }
         }
+
+        processUsings(objs);
+
         // process enum in models
         return postProcessModelsEnum(objs);
     }
 
+    private void processUsings(Map<String, Object> objs) {
```

Comment:
> I know @wing328 was working on a solution to fully qualify models but I don't know the state of that work.

Last time when I tried, it's not that straightforward due to issue with Array/Map of models (with fully-qualified names).